### ISSUE TITLE
[Source Request]: Halton United Kingdom

### ISSUE THREAD

**Issue by RegPhenna** [mampfes/hacs_waste_collection_schedule#2493]
### Municipality / Region

Halton, United Kingdom

### Collection Calendar Webpage

https://webapp.halton.gov.uk/PublicWebForms/WasteServiceSearchv1.aspx#collections

### Example Address

9 hawkhead close Rucorn wa7 3je

### Collection Data Format

As PDF Download

**Comment by 5ila5** [comment]
I don't think we can implement it as they use reCAPTCHA before showing the collection dates.

I think you're stuck using the [static source](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/static.md)

**Comment by markvp** [comment]
Closing as duplicate of #2137.
